libs/designsystem/shared/src/translation/translation.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { Inject, LOCALE_ID } from '@angular/core';
+import { da } from './translations/da';
+import { en } from './translations/en';
+import { Translation } from './translation.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TranslationService {
+  private translations: { [key: string]: Translation } = {
+    da,
+    en,
+  };
+
+  public currentTranslation: Translation = en;
+
+  constructor(@Inject(LOCALE_ID) private localeId: string) {
+    this.setCurrentTranslation(localeId);
+  }
+
+  setCurrentTranslation(localeId: string): string {
+    const baseLocaleId = localeId.split('-')[0];
```

Comment:
Don't know if destructuring is more readable 🤔 
```suggestion
    const [baseLocaleId] = localeId.split('-');
```

Reply:
In this case I think it reads a bit better that you can go "split string and take the first substring" rather than having to understand the destructuring part 🤔 So I'll leave it as is if ok.